CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -850,9 +850,6 @@ CF_INLINE const char *CFPathRelativeToAppleFrameworksRoot(const char *path, Bool
 #if (DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_EMBEDDED_MINI || DEPLOYMENT_TARGET_LINUX)
 #if __HAS_DISPATCH__
 
-#include <dispatch/dispatch.h>
-#include <dispatch/private.h>
```

Comment:
There's a workaround for dispatch/private.h in the pending overlay pull requests.  Basically it exploits the symlink that is made by configure from $(top_bindir)/tests/dispatch to $(top_srcdir)/private to allow an include of dispatch/private.h to succeed.  Personally, I think this is a bit quirky, but it works (and is consistent with how the include of dispatch/private.h is made to work when compiling libdispatch's test cases on linux).